app/screens/conversion-flow/use-convert-money-details.ts

```diff
@@ -0,0 +1,95 @@
+import { DisplayCurrency, MoneyAmount, WalletOrDisplayCurrency } from "@app/types/amounts"
+import React from "react"
+import { usePriceConversion } from "../../hooks/use-price-conversion"
+import { WalletCurrency } from "@app/graphql/generated"
+
+type Wallet = {
+  balance: number
```

Comment:
I'm wondering whether we should introduce branded type on the mobile end

Reply:
For balance?

Reply:
yeah for instance. maybe those should be branded at the graphql layer. just speaking out loud here not a strong opinion on this

Reply:
I think that would be awesome, but I'm not sure how that works. They are currently branded scalars coming from the graphql api and then converted to primitive types.